libfreerdp/core/transport.c

```diff
@@ -857,12 +857,14 @@ int transport_check_fds(rdpTransport* transport)
 	int status;
 	int recv_status;
 	wStream* received;
-	DWORD now = GetTickCount();
+	DWORD now = 0;
```

Comment:
Why did you remove GetTickCount here? This breaks the now < dueDate check in the loop condition ...